apps/andi/lib/andi/input_schemas/organization.ex

```diff
@@ -64,7 +64,13 @@ defmodule Andi.InputSchemas.Organization do
     id = Ecto.Changeset.get_field(changeset, :id)
     org_name = Ecto.Changeset.get_field(changeset, :orgName)
 
-    case Organizations.is_unique?(id, org_name) do
+    safe_org_name =
+      case org_name do
+        nil -> ""
+        x -> x
+      end
+
+    case Organizations.is_unique?(id, safe_org_name) do
```

Comment:
Instead of manipulating the input parameter, can we create a hyrdra head to `Organizations.is_unique?`

```elixir
# Organizations schema file or wherever this function is implemented
# ... other `is_unique?`...

def is_unique?(_id, nil) do
  false
end

# ...
```

Reply:
Nice, I like it. I'll make that change